packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/splaLicence/translations/Messages_fr_FR.json

```
@@ -0,0 +1,3 @@
+{
+  "dedicatedCloud_datacenter_backup_nospla": "Votre Private Cloud doit avoir une licence SPLA avant de pouvoir activer le backup. Afin d'activer la licence SPLA, veuillez vous rendre dans la section <a data-ui-sref=\"{{ licenceRoute }}\">Licence Windows</a> de votre produit."
```

What I meant by injected was that we should avoid `ui-sref` usage and providde the link to the component which will use a `href`

---

I am not very clear about what you are asking. Is it possible for you to refer me to a page where this is used?

---

@radireddy something like we usually do when it is related to `ui-router`
Something like this in the routing : https://github.com/ovh/manager/blob/develop/packages/manager/modules/pci/src/projects/project/instances/instances.routing.js#L118
so you would have 
```suggestion
  "dedicatedCloud_datacenter_backup_nospla": "Votre Private Cloud doit avoir une licence SPLA avant de pouvoir activer le backup. Afin d'activer la licence SPLA, veuillez vous rendre dans la section <a href=\"{{ licenseRoute }}\">Licence Windows</a> de votre produit."
```
with the component having a binding like this : 
```suggestion
bindings: {
  licenseRoute: '<'
 }
```